### PR TITLE
#160945059 ft- consuming get all menu

### PR DESCRIPTION
**Pull request by fejiroofficial**
**What does this PR do?**
This is a pull request for:
- consuming the API endpoint `GET`  `/api/v1/menu`
- adding a little responsiveness to app's homepage.

**Description of Task to be completed?**
- fetch API from `https://food-fast-app.herokuapp.com/api/v1/menu`
- Consume api such that a user can be able to see food on menu.

**How should this be manually tested?**
- Clone the repo using git clone` https://github.com/fejiroofficial/Fast-Food-Fast-app.git.`
- run `git checkout ft-get-menu-160945059`
- `using postman` `https://food-fast-app.herokuapp.com/api/v1/menu`
- [gh-pages](https://fejiroofficial.github.io/Fast-Food-Fast-app/UI/index.html)

**Relevant pivotal stories**
[160945059](https://www.pivotaltracker.com/n/projects/2196470/stories/160945059)

### CODE REVIEW

File: UI/js/getMenu.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+window.addEventListener('load', (event) => {
+  event.preventDefault();
+  const meal = document.querySelector('div.meal-box-content');
+
+  const load = (foodMenu) => {
+    foodMenu.forEach((element) => {
+      const itemName = element.item_name;
+      const price = element.price;
+      const foodUrl = element.food_url;
+      const template = `
```
Comment (ChidinmaOrajiaku) [inline UI/js/getMenu.js]: I don’t think template is a good naming convention for this. Please refactor

File: UI/js/getMenu.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  fetch('https://food-fast-app.herokuapp.com/api/v1/menu', {
+    method: 'get',
+    headers: {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
+    },
+  })
+    .then((response) => {
+      response.json()
+        .then((data) => {
```
Comment (ChidinmaOrajiaku) [inline UI/js/getMenu.js]: `data` is a very ambiguous name.

File: UI/js/getMenu.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          const foodMenu = data.menu;
+
+          load(foodMenu);
+        })
+        .catch(err => console.log(err));
```
Comment (ChidinmaOrajiaku) [inline UI/js/getMenu.js]: Why are you logging the error on the console? Is it not better to throw than log?

**Review by ChidinmaOrajiaku — COMMENTED** [review]
Also, no `console.log` or commented codes on github

**Review by ChidinmaOrajiaku — CHANGES_REQUESTED** [review]
Also, no `console.log`  on github

File: UI/js/getMenu.js (hunk 3 above)
Comment (fejiroofficial) [reply UI/js/getMenu.js]: done

File: UI/js/getMenu.js (hunk 2 above)
Comment (fejiroofficial) [reply UI/js/getMenu.js]: done

File: UI/js/getMenu.js (hunk 1 above)
Comment (fejiroofficial) [reply UI/js/getMenu.js]: done

**Comment by ChidinmaOrajiaku** [comment]
LGTM. Fix your merge conflict

**Pull request merged**